toonz/sources/toonz/menubar.cpp

```diff
@@ -1253,6 +1253,7 @@ QMenuBar *StackedMenuBar::createFullMenuBar() {
   addMenuItem(xsheetMenu, MI_MergeCmapped);
   xsheetMenu->addSeparator();
   addMenuItem(xsheetMenu, MI_MergeColumns);
+  addMenuItem(xsheetMenu, MI_MergeColumnsWOGroups);
```

Comment:
> I've also added two commands that bypass the dialog box:`Merge Levels (Do Not group Vector Levels)` and `Merge Levels (Group Vector Levels)`. I did not add them to the default menubar--they're for anyone who might want to keybind them.

This makes the command show on the menubar by default. Will definitely show on macOS version.
 Should this be removed?